transport/factory.py

```diff
@@ -0,0 +1,16 @@
+from transport.matrix.layer import MatrixLayer as MatrixTransportLayer
+from transport.rif_comms.layer import RifCommsLayer as RifCommsTransportLayer
+
+
+class Factory:
+    """
+    Factory is a class used to choose between transport layer implementations for Lumino and create an instance.
+    """
+
+    @staticmethod
+    def create(transport_type : str, config : dict):
```

Comment:
we can add a type hint for the returned value here

Reply:
done